packages/components/tree-select/src/tree.ts

```diff
@@ -1,6 +1,8 @@
 // @ts-nocheck
 import { computed, nextTick, toRefs, watch } from 'vue'
 import { isEqual, isNil, pick } from 'lodash-unified'
+import { useEventListener } from '@vueuse/core'
+import { EVENT_CODE } from 'element-plus'
```

Comment:
importing directly from 'element-plus' can cause local build errors. [ref](https://github.com/element-plus/element-plus/pull/20415)

```suggestion
import { EVENT_CODE } from '@element-plus/constants'
```

Reply:
done